scss/_card.scss

```diff
@@ -131,6 +131,21 @@
   @include border-bottom-radius($card-inner-border-radius);
 }
 
+.card-img-right {
+  height: 100%; // Required because we use flexbox and this inherently applies align-self: stretch
+  @include border-right-radius($card-inner-border-radius);
+}
+
+.card-img-left {
+  height: 100%; // Required because we use flexbox and this inherently applies align-self: stretch
+  @include border-left-radius($card-inner-border-radius);
+}
+
+
+// Horizontal cards
+.card-horizontal {
+  flex-direction: row;
```

Comment:
Hi @gijsbotje This could have been a clever and simple solution, but you have to remember that body and image are not the only elements a card can have. For example, if you add a `card-footer` it will go to the right and it'll look really bad.
Although it had potential I don't think it's general and flexible enough.

Reply:
@andresgalante it is exact my simple mind came across.

Reply:
Not elegant, and NOT how it is supposed to.
Given from the Template example, using inside the card columns made the `card-footer`
at least working:
        `<div class="col-md-6">
			<div class="card mb-4 box-shadow ">
				<div class="row">
					<div class="col-md-8">
						<div class="card-body flex-column align-items-start">
							<strong class="d-inline-block mb-2 text-primary">World</strong>
							<h3 class="mb-0">
							<a class="text-dark" href="#">Featured post</a>
							</h3>
							<div class="mb-1 text-muted">Nov 12</div>
							<p class="card-text mb-auto">This is a wider card with supporting text below as a natural lead-in to additional content.</p>
							<a href="#">Continue reading</a>
						</div>
					</div>
					<div class="col-md-4">            
						<img class="card-img-right flex-auto d-none d-lg-block" data-src="holder.js/150x150?theme=thumb" alt="Card image cap">
					</div> 			
				</div>
				<div class="card-footer">test</div>			
			</div>
        </div>`